app.js

```diff
@@ -109,7 +111,7 @@ function getMatchedProduct(allProducts, productToBeAdded) {
 
 function addProductToCart(product) {
   let result = cartContainer.innerHTML;
-  result += `<div class="cart-item">
+  result += `<div class="cart-item" data-id="${product.id}">
```

Comment:
FYI, adding ids to products was not required in this task but I did anyways so that you can work on your next ticket of removing items with the help of product id.

Reply:
@SabaMallah

Reply:
Nice!